src/admin/content-block/components/BlockKlaar/BlockKlaar.tsx

```diff
@@ -16,6 +16,6 @@ export const BlockKlaar: FunctionComponent<BlockKlaarProps> = ({ className, date
 			<span>KLAAR</span>
```

Comment:
Vergeet dit niet toe te passen op de components-versie. Ik heb namelijk in mijn PR deze file al verwijderd en de components-versie in gebruik genomen. 😄

Reply:
het is niet kritiek aangezien deze error zich enkel voordeed bij de oude block state van een klaar block